### ISSUE TITLE
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'accessibility.settings' doesn't exist (Co...

### ISSUE THREAD

**Issue by jobara** [accessibility-in-action/platform#2044]
**`Illuminate\Database\QueryException`**

SQLSTATE[42S02]: Base table or view not found: 1146 Table 'accessibility.settings' doesn't exist (Connection: mysql, SQL: select `name`, `payload` from `settings` where `group` = general)

:dart: Illuminate\Database\Connection::runQueryCallback
:page_facing_up: /vendor/laravel/framework/src/Illuminate/Database/Connection.php

```php
799   }
800   
801*  throw new QueryException(
802       $this->getName(), $query, $this->prepareBindings($bindings), $e
803   );
```
[Open in Flare](https://flareapp.io/errors/4766877/latest)

**Comment by jobara** [comment]
May also be related to #2042

**Comment by jobara** [comment]
This particular issue has not re-occurred recently and was likely fixed by #2134
